data/pug.txt

```diff
@@ -192,6 +195,7 @@ outfit "Pug Gridfire Turret"
 		"hull damage" 600
 		"blast radius" 40
 		"piercing" .3
+	description "No one quite knows how the Gridfire Turret works, and the Pug aren't interested in explaining it."
```

Comment:
"Quite knows" is a weird expression. Simply "understands" sounds better to me. Also, I would cut out the "it" at the end.

Reply:
Done both corrections